modules/cnf-node-tuning-operator-disabling-cpu-load-balancing-for-dpdk.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-node-tuning-operator-disabling-cpu-load-balancing-for-dpdk_{context}"]
+= Disabling CPU load balancing for DPDK
+
+Functionality to disable or enable CPU load balancing is implemented on the CRI-O level. The code under the CRI-O disables or enables CPU load balancing only when the following requirements are met.
+
+* The pod must use the `performance-<profile-name>` runtime class. You can get the proper name by looking at the status of the performance profile, as shown here:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+...
+status:
+  ...
+  runtimeClass: performance-manual
+----
+
+[NOTE]
+====
+Currently, disabling CPU load balancing is not supported with cgroup v2.
+====
+
+The Node Tuning Operator is responsible for the creation of the high-performance runtime handler config snippet under relevant nodes and for creation of the high-performance runtime class under the cluster. It will have the same content as default runtime handler except it enables the CPU load balancing configuration functionality.
+
+To disable the CPU load balancing for the pod, the `Pod` specification must include the following fields:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  ...
+  annotations:
+    ...
+    cpu-load-balancing.crio.io: "disable"
+    ...
+  ...
+spec:
+  ...
+  runtimeClassName: performance-<profile_name>
+  ...
+----
+
+[NOTE]
+====
+Only disable CPU load balancing when the CPU manager static policy is enabled and for pods with guaranteed QoS that use whole CPUs. Otherwise, disabling CPU load balancing can affect the performance of other containers in the cluster.
```

Comment:
It is accurate. Because all the conditions you talk about are satisfied even in the unlikely case where the cpu manager policy is set to disabled (= no pinning!).